### ISSUE TITLE
_get_unicode with force=True breaks with unicode characters on Python 2.7

### ISSUE THREAD

**Issue by ArthurPBressan** [influxdata/influxdb-python#268]
On Python 2.7, the `_get_unicode` function on `line_protocol.py` breaks if the passed text contains an Unicode code point with a `UnicodeEncodeError` if `force` is True. 

For example:
```python
Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
Type "copyright", "credits" or "license" for more information.

In [1]: from influxdb import line_protocol

In [2]: line_protocol._get_unicode(u'ç')
Out[2]: u'\xe7'

In [3]: line_protocol._get_unicode(u'ç', force=True)
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-3-fb06071b454e> in <module>()
----> 1 line_protocol._get_unicode(u'ç', force=True)

/<virtualenv>/local/lib/python2.7/site-packages/influxdb/line_protocol.pyc in _get_unicode(data, force)
     75         return ''
     76     elif force:
---> 77         return str(data)
     78     else:
     79         return data
```
I worked around it by substituting `return str(data)` with `return unicode(data)`. I can't really say what is the full impact of this change, but this seemed to work so far.

This problem shows up when I try to generate a line when one of its tags has a string with Unicode characters:

```python
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/client.py", line 373, in write_points
    tags=tags)
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/client.py", line 418, in _write_points
    expected_response_code=204
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/client.py", line 258, in write
    data=make_lines(data, precision).encode('utf-8'),
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/line_protocol.py", line 108, in make_lines
    value = _escape_tag(tags[tag_key])
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/line_protocol.py", line 40, in _escape_tag
    tag = _get_unicode(tag, force=True)
  File "<virtualenv>/local/lib/python2.7/site-packages/influxdb/line_protocol.py", line 77, in _get_unicode
    return str(data)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 9: ordinal not in range(128)
```

**Comment by pengyao** [comment]
+1

**Comment by pengyao** [comment]
#296

**Comment by aviau** [comment]
Closed in #296
